backend/session/logind.c

```diff
@@ -307,6 +341,107 @@ static int resume_device(sd_bus_message *msg, void *userdata, sd_bus_error *ret_
 	return 0;
 }
 
+static int properties_changed(sd_bus_message *msg, void *userdata,
+		sd_bus_error *ret_error) {
+	struct logind_session *session = userdata;
+	int ret = 0;
+
+	// if we have drm fd we don't depend on this
+	if (session->has_drm) {
+		return 0;
+	}
+
+	// PropertiesChanged 1: interface
+	const char *interface;
+	ret = sd_bus_message_read_basic(msg, 's', &interface); // skip path
+	if (ret < 0) {
+		wlr_log(WLR_ERROR, "Failed to parse D-Bus PropertiesChanged (1) %s",
```

Comment:
I don't think these numbers are necessary. The line number is enough to distingush these calls.

Reply:
agreed